core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -312,4 +326,215 @@ private MetricsReporter metricsReporter() {
 
     return metricsReporter;
   }
+
+  protected ViewOperations newViewOps(TableIdentifier viewIdentifier) {
+    throw new UnsupportedOperationException("Not implemented: newViewOps");
+  }
+
+  @Override
+  public List<TableIdentifier> listViews(Namespace namespace) {
+    throw new UnsupportedOperationException("Not implemented: listViews");
+  }
+
+  @Override
+  public View loadView(TableIdentifier identifier) {
+    throw new UnsupportedOperationException("Not implemented: loadView");
+  }
+
+  @Override
+  public boolean dropView(TableIdentifier identifier) {
+    throw new UnsupportedOperationException("Not implemented: dropView");
+  }
+
+  @Override
+  public void renameView(TableIdentifier from, TableIdentifier to) {
+    throw new UnsupportedOperationException("Not implemented: renameView");
+  }
+
+  @Override
+  public ViewBuilder buildView(TableIdentifier identifier) {
+    return new BaseViewBuilder(identifier);
+  }
+
+  protected class BaseViewBuilder implements ViewBuilder {
+    private final TableIdentifier viewIdentifier;
+    private final ImmutableViewVersion.Builder viewVersionBuilder = ImmutableViewVersion.builder();
+    private ImmutableSQLViewRepresentation.Builder viewRepresentationBuilder;
+    private final ImmutableViewMetadata.Builder builder = ImmutableViewMetadata.builder();
+
+    public BaseViewBuilder(TableIdentifier viewIdentifier) {
```

Comment:
It's weird to have such a long variable name when it conflicts with the type name. Normally we'd use simply `ident` or `identifier` because we prefer shorter names if the extra words are not adding clarity.